src/main/java/uk/gov/digital/ho/hocs/audit/export/ExportService.java

```diff
@@ -48,6 +50,7 @@
     public static final String[] TOPIC_EVENTS = {"CASE_TOPIC_CREATED", "CASE_TOPIC_DELETED"};
     public static final String[] CORRESPONDENT_EVENTS = {"CORRESPONDENT_DELETED", "CORRESPONDENT_CREATED", "CORRESPONDENT_UPDATED"};
     public static final String[] ALLOCATION_EVENTS = {"STAGE_ALLOCATED_TO_TEAM", "STAGE_CREATED", "STAGE_RECREATED", "STAGE_COMPLETED", "STAGE_ALLOCATED_TO_USER", "STAGE_UNALLOCATED_FROM_USER"};
+    private final AtomicReference<String> teamList = new AtomicReference<>();
```

Comment:
The more I think about it - is this required? Having a string in the method would effectively do the same thing as it's not static and other threads cannot interact with it?

Reply:
I've changed the way the for loop worked for teams and removed this atomic reference, it was needed before due to the forEach loop